Find similar incidents bug (#11560)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_3_23.md

#### Scripts
##### FindSimilarIncidents

> - Fixed an issue where the script did not find similar incidents when using **similarIncidentFields** argument from type int.

Reviewer: ```suggestion
- Fixed an issue where the script would not find similar incidents while using the **similarIncidentFields** argument when given an int type.
```
